snap/hooks/configure

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
```

Comment:
I didn't saw configuration being used anywhere (e.g. `snapctl get postgres.port`) 
So you can probably remove this file